src/conformance.md

```diff
@@ -0,0 +1,292 @@
+<p style="color:red;font-size:50px;font-style:oblique">
+.. DRAFT .. DRAFT .. DRAFT ..
+</p>
+
+***Status : first draft***
+<br>_Hopefully, this can evolve into an agreed final form
+by the time of the next UGRID release._
+
+# Conformance rules for UGRID Conventions v1.x
+
+This information is intended to follow the practice established by the
+[CF conformance pages][cf_conformance], as attached to the [CF conventions][cf_conventions].
+
+In that spirit, this is likewise "intended to be a concise summary" of the
+main convention, and "if there are any discrepancies ... the conventions document
+is the ultimate authority."
+ 
+Each reference statement is provided with a short code :
+"Rxxx" for requirements and "Axxx" for advisory recommendations,
+in the manner of the [Flake8 error and warning codes][flake8_codes], and the [PyCodeStyle ones][pycodestyle_codes].
+<br>Future editions will maintain these reference numbers, but possibly add new ones.  They can be used as definitive
+references by a conformance-checker utility. [^10]
+
+**This version** is consistent with :
+
+  * UGRID v1.0 
+  * [CF version 1.9][cf_current_version]
```

Comment:
Update to 1.10 ?

Reply:
Hmm -- UGRID just got added to CF, specifically with 1.0 -- so let's keep 1.0